package.json

```diff
@@ -151,9 +152,9 @@
     "linkify-it": "^2.0.3",
     "loader-utils": "^1.0.2",
     "prepend-http": "1.0.4",
-    "react": "^15.5.4",
+    "react": "^16.9.0",
```

Comment:
so here's 16.9, but up there we have shouldn't we use 16.3 here to be in line with the plugins

Reply:
Latest version shows warnings about using legacy lifecycle hook which should be fixed.